src/style/app.module.css

```diff
@@ -1942,185 +2058,794 @@ input[type='radio']:checked + label:hover {
   color: var(--black-color) !important;
 }
 
-.dropdownToggle {
-  margin-bottom: 0;
+.dropdownToggle {
+  margin-bottom: 0;
+  display: flex;
+}
+
+.dropdownModalToggle {
+  width: 50%;
+}
+
+.greenregbtn {
+  margin-top: 1rem;
+  border: 1px solid var(--bs-gray-300);
+  box-shadow: 0 2px 2px var(--bs-gray-300);
+  padding: 10px 10px;
+  border-radius: 5px;
+  background-color: var(--bs-primary);
+  width: 100%;
+  font-size: 16px;
+  color: var(--bs-white);
+  outline: none;
+  font-weight: 600;
+  cursor: pointer;
+  transition:
+    transform 0.2s,
+    box-shadow 0.2s;
+  width: 100%;
+}
+
+.manageBtn {
+  margin: 1rem 0 0;
+  margin-top: 15px;
+  border: 1px solid var(--grey-border-box-color);
+  box-shadow: 0 2px 2px var(--grey-border-box-color);
+  padding: 10px 10px;
+  border-radius: 5px;
+  font-size: 16px;
+  color: var(--bs-white);
+  outline: none;
+  font-weight: 600;
+  cursor: pointer;
+  width: 45%;
+  transition:
+    transform 0.2s,
+    box-shadow 0.2s;
+}
+
+.removeFilterIcon {
+  cursor: pointer;
+}
+
+.searchForm {
+  display: inline;
+}
+
+.view {
+  margin-left: 2%;
+  font-weight: 600;
+  font-size: 16px;
+  color: var(--bs-gray-600);
+}
+
+/* header (search, filter, dropdown) */
+.btncon .btnsContainer {
+  display: flex;
+  margin: 0.5rem 0 1.5rem 0;
+}
+
+.btncon .btnsContainer .input {
+  flex: 1;
+  min-width: 18rem;
+  position: relative;
+}
+
+.btncon .btnsContainer input {
+  outline: 1px solid var(--bs-gray-400);
+}
+
+.btncon .btnsContainer .input button {
+  width: 52px;
+}
+
+.noOutline input {
+  outline: none;
+}
+
+.noOutline input:disabled {
+  -webkit-text-fill-color: var(--black-color) !important;
+}
+
+.noOutline textarea:disabled {
+  color: var(--black-color) !important;
+  opacity: 1;
+}
+
+.inputFields {
+  box-shadow: 0 1px 1px var(--brand-primary);
+}
+
+.dropdowns {
+  background-color: var(--bs-white);
+  border: 1px solid var(--dropdown-border-color);
+  position: relative;
+  display: inline-block;
+  color: black;
+}
+
+.chipIcon {
+  height: 0.9rem !important;
+}
+
+.chip {
+  height: 1.5rem !important;
+  margin: 0.15rem 0 0 1.25rem;
+}
+
+.active {
+  background-color: var(--status-active-bg);
+  color: var(--light-blue) !important;
+  border-color: var(--light-blue) !important;
+}
+
+.pending {
+  background-color: var(--status-pending-bg);
+  color: var(--light-blue) !important;
+  border-color: var(--light-blue) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`